File: fingertable.cpp
#include "node.h"

void FingerTable::set(size_t index, Node* successor) {
	fingerTable_[index] = successor;
}

void FingerTable::get(size_t index) {
	return fingerTable_[index];
}

void FingerTable::prettyprint() {

};

File: src/Chord/FingerTable.java
package Chord;

class FingerTable {
    private int hostNodeId;

    FingerTable(Node node) {
        this.hostNodeId = node.getId();
    }

    FingerTable(int hostNodeId) {
        this.hostNodeId = hostNodeId;
    }

    public int getHostNodeId() {
        return hostNodeId;
    }

    protected void prettyPrint() {

    }
}